core/src/main/java/org/apache/brooklyn/core/mgmt/persist/DeserializingClassRenamesProvider.java

```diff
@@ -72,7 +81,7 @@
             } finally {
                 Streams.closeQuietly(resource);
             }
-        } else {
+        } catch (Exception e) {
```

Comment:
This one is safe. It is for the `new ResourceUtils(DeserializingClassRenamesProvider.class).getResourceFromUrl(url);`

As per as the javadoc:
> Takes a string which is treated as a URL (with some extended "schemes" also expected), or as a path to something either on the classpath (absolute only) or the local filesystem (relative or absolute, depending on leading slash)
URLs can be of the form classpath://com/acme/Foo.properties as well as file:///home/... and http://acme.com/....
**Throws exception if not found**, using the context parameter passed into the constructor.
TODO may want OSGi, or typed object; should consider pax url
Returns:
a stream, or throws exception **(never returns null)**